src/BoltOn/BoltOn.csproj

```diff
@@ -29,6 +29,7 @@
         <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
         <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
         <PackageReference Include="NuGet.Build.Packaging" Version="0.2.0" />
+        <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
```

Comment:
why do we need this in the core project?

Reply:
@gokulm : removed it.